docs/quick-start.md

```diff
@@ -211,7 +199,7 @@ a cluster, as described in the [programming guide](programming-guide.html#initia
 It may seem silly to use Spark to explore and cache a 100-line text file. The interesting part is
 that these same functions can be used on very large data sets, even when they are striped across
 tens or hundreds of nodes. You can also do this interactively by connecting `bin/pyspark` to
-a cluster, as described in the [programming guide](programming-guide.html#initializing-spark).
+a cluster, as described in the [programming guide](rdd-programming-guide.html#initializing-spark).
```

Comment:
we probably wanna separate the previous programming guide into a basic programming guide and an RDD programming guide. The basic programming guide should talk about some basic knowledge like how to initialize spark, how to deploy spark, etc.